pkg/rm/tcc/tcc_resource.go

```diff
@@ -90,15 +90,18 @@ type TCCResourceManager struct {
 }
 
 // register transaction branch
-func (t *TCCResourceManager) BranchRegister(ctx context.Context, branchType branch.BranchType, resourceId, clientId, xid, applicationData, lockKeys string) (int64, error) {
-	return t.rmRemoting.BranchRegister(branch.BranchTypeTCC, resourceId, clientId, xid, applicationData, lockKeys)
+func (t *TCCResourceManager) BranchRegister(ctx context.Context, param rm.BranchRegisterParam) (int64, error) {
+	return t.rmRemoting.BranchRegister(param)
 }
 
-func (t *TCCResourceManager) BranchReport(ctx context.Context, branchType branch.BranchType, xid string, branchId int64, status branch.BranchStatus, applicationData string) error {
-	return t.rmRemoting.BranchReport(branchType, xid, branchId, status, applicationData)
+// report status of transaction branch
+func (t *TCCResourceManager) BranchReport(ctx context.Context, param rm.BranchReportParam) error {
+	return t.rmRemoting.BranchReport(param)
+
 }
 
-func (t *TCCResourceManager) LockQuery(ctx context.Context, ranchType branch.BranchType, resourceId, xid, lockKeys string) (bool, error) {
+// query lock status of transaction branch
+func (t *TCCResourceManager) LockQuery(ctx context.Context, param rm.LockQueryParam) (bool, error) {
```

Comment:
comment add function name at first， as
// LockQuery query lock status of transaction branch